src/extension/neoExpress/neoExpress.ts

```diff
@@ -43,18 +44,28 @@ export default class NeoExpress {
     this.checkForDotNetPassedAt = 0;
   }
 
-  runInTerminal(name: string, command: Command, ...options: string[]) {
+  async runInTerminal(name: string, command: Command, ...options: string[]) {
     if (!this.checkForDotNet()) {
       return null;
     }
     const dotNetArguments = [this.binaryPath, command, ...options];
-    const terminal = vscode.window.createTerminal({
-      name,
-      shellPath: this.dotnetPath,
-      shellArgs: dotNetArguments,
-      hideFromUser: false,
+    const pty = new NeoExpressTerminal(this.dotnetPath, dotNetArguments);
+    const terminal = vscode.window.createTerminal({ name, pty });
+
+    const hasStarted: Promise<void> = new Promise((resolve) => {
+      pty.onDidWrite((data) => {
+        if (data.indexOf("OnStart") !== -1) {
```

Comment:
File an issue in the neo express repo and I can add some standard text to `ExpressChainManager.RunAsync` to emit a string that neo-express controls instead of depending on log text generated by consensus plugin

Reply:
See neo-project/neo-express#123